src/Sylius/Bundle/AdminBundle/Resources/assets/scripts/apexcharts.js

```diff
@@ -0,0 +1,99 @@
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+import ApexCharts from 'apexcharts';
+
+(function () {
+  const options = {
+    series: [{
+      name: 'Inflation',
+      data: [2.3, 3.1, 4.0, 10.1, 4.0, 3.6, 3.2, 2.3, 1.4, 0.8, 0.5, 0.2],
+    }],
+    chart: {
+      height: 350,
+      type: 'bar',
+    },
+    plotOptions: {
+      bar: {
+        borderRadius: 10,
+        dataLabels: {
+          position: 'top', // top, center, bottom
+        },
+      },
+    },
+    dataLabels: {
+      enabled: true,
+      formatter(val) {
+        return `${val}%`;
+      },
+      offsetY: -20,
+      style: {
+        fontSize: '12px',
+        colors: ['#304758'],
+      },
+    },
+
+    xaxis: {
+      categories: ['Jan', 'Feb', 'Mar', 'Apr', 'May', 'Jun', 'Jul', 'Aug', 'Sep', 'Oct', 'Nov', 'Dec'],
+      position: 'top',
+      axisBorder: {
+        show: false,
+      },
+      axisTicks: {
+        show: false,
+      },
+      crosshairs: {
+        fill: {
+          type: 'gradient',
+          gradient: {
+            colorFrom: '#D8E3F0',
+            colorTo: '#BED1E6',
+            stops: [0, 100],
+            opacityFrom: 0.4,
+            opacityTo: 0.5,
+          },
+        },
+      },
+      tooltip: {
+        enabled: true,
+      }
+    },
+    yaxis: {
+      axisBorder: {
+        show: false,
+      },
+      axisTicks: {
+        show: false,
+      },
+      labels: {
+        show: false,
+        formatter(val) {
+          return `${val}%`;
+        },
+      },
+
+    },
+    title: {
+      floating: true,
+      offsetY: 330,
+      align: 'center',
+      style: {
+        color: '#444',
+      },
+    },
+  };
+
+  // eslint-disable-next-line no-undef
+  const ch = document.querySelector('#chart-demo');
+  if (ch) {
+    // eslint-disable-next-line no-undef
+    const chart = new ApexCharts(document.querySelector('#chart-demo'), options);
```

Comment:
Shouldn't this part of attaching chart to element be at `app.js` as it was in previous version?